libraries/joomla/form/rules/equals.php

```
@@ -44,13 +44,13 @@ public function test($element, $value, $group = null, $input = null, $form = nul
 		// Check that a validation field is set.
 		if (!$field)
 		{
-			return new JException(JText::sprintf('JLIB_FORM_INVALID_FORM_RULE', get_class($this)));
```

I think this should "throw" and UnexpectedValueException (there is an error in the XML - do not pass Go or collect $200).